src/main/java/com/github/sommeri/less4j/core/output/SourceMapBuilder.java

```diff
@@ -24,6 +26,7 @@
   private final SourceMapGenerator generator;
   private final LessCompiler.SourceMapConfiguration configuration;
   private final Collection<LessSource> additionalSourceFiles;
+  private final Map<LessSource, String> cachedContent;
```

Comment:
I think it would be better to use ConcurrentMap with `putIfAbsent()` instead of `put()`.